src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -462,7 +464,7 @@ function EmojiPickerMenu(props) {
                 />
             );
         },
-        [isUsingKeyboardMovement, highlightedIndex, onEmojiSelected, preferredSkinTone, translate, highlightFirstEmoji],
+        [preferredSkinTone, highlightedIndex, isUsingKeyboardMovement, highlightFirstEmoji, styles.emojiHeaderContainer, styles.textLabelSupporting, translate, onEmojiSelected],
```

Comment:
you can save some equality checks by adding just `styles` there - this whole object gets recreated each time and stays stable referentially for the whole time until we change the 'theme' again.